contributing/index.md

```diff
@@ -77,6 +83,9 @@ You can deactivate the virtual environment again using:
 deactivate
 ```
 
+_Note: Don't forget to activate your virtual environment every time you wish to work on vorta._
```

Comment:
nit: Vorta instead of vorta (for consistency)?

Reply:
@diivi I think the readers personal `vorta` repository/directory is being referred to in this instance, rather than Vorta the project as a whole.